lib/carrierwave/storage/aws_file.rb

```diff
@@ -60,6 +60,10 @@ def copy_to(new_path)
         bucket.object(new_path).copy_from(copy_source: "#{bucket.name}/#{file.key}")
       end
 
+      def signed_url(options = {})
+        uploader.sign_urls[:signer].call(public_url, options)
```

Comment:
I'd prefer not to rely on `try` for defensiveness. The `add_config` macro simply adds getter and setter methods on the instance and the class. For `aws_acl` I simply added each of the methods manually. That seems like the way to go here. You avoid having the proc called during instantiation and gain the opportunity for additional validation.